src/pathfinding_service/exceptions.py

```diff
@@ -1,6 +1,10 @@
 from typing import Any, Dict, Optional
 
 
+class MessageException(Exception):
```

Comment:
This name could be more descriptive. Like `InvalidMessage` or `InvalidBalanceUpdate`.

Reply:
I would go for `InvalidCapacityUpdate` ?!